.github/.kubescape/exceptions.json

```diff
@@ -14,5 +14,167 @@
         }
       }
     ]
+  },
+  {
+    "name": "application-credentials-in-configuration-files",
+    "policyType": "postureExceptionPolicy",
+    "actions": [
+      "alertOnly"
+    ],
+    "resources": [
+      {
+        "designatorType": "Attributes",
+        "attributes": {
+          "kind": "Deployment"
+        }
+      }
+    ],
+    "posturePolicies": [
+        {
+            "controlID": "C-0012"
+        }
+    ]
+  },
+  {
+    "name": "images-allowed-registry",
```

Comment:
instead of disabling this rule, you could add the missing registry to the allowed list. see [link](https://hub.armosec.io/docs/c-0078)